python/core/geometry/qgspoint.sip

```diff
@@ -16,53 +16,42 @@ class QgsPoint: QgsAbstractGeometry
  Point geometry type, with support for z-dimension and m-values.
 .. versionadded:: 3.0
 %End
-
 %TypeHeaderCode
 #include "qgspoint.h"
 %End
   public:
+//    static double NaN; // = std::numeric_limits<double>::quiet_NaN();
 
-    QgsPoint( double x = 0.0, double y = 0.0, double z = 0.0, double m = 0.0 );
+    QgsPoint( double x = 0.0, double y = 0.0, SIP_PYOBJECT z = Py_None, SIP_PYOBJECT m = Py_None ) [( double x = 0.0, double y = 0.0, double z = 0.0, double m = 0.0 )];
```

Comment:
well I would do it within an `#ifdef SIP_RUN` otherwise we could use something like:
`QgsPoint( double x = 0.0, double y = 0.0, double z = 0.0, double m = 0.0 ) SIP_SIGNATURE( double x = 0.0, double y = 0.0, SIP_PYOBJECT z = Py_None, SIP_PYOBJECT m = Py_None );`